docs/component-stability.md

```diff
@@ -66,6 +66,74 @@ Stable components MUST be compatible between minor versions unless critical secu
 component owner MUST provide a migration path and a reasonable time frame for users to upgrade. The same rules from beta
 components apply to stable when it comes to configuration changes.
 
+#### Observability requirements
```

Comment:
You mention a couple of times the concept of items. It may make sense to define this first at the beginning of this section so you don't have to explain it multiple times